### ISSUE TITLE
Python 3: NameError: name 'long' is not defined

### ISSUE THREAD

**Issue by jedie** [edoburu/django-oscar-docdata#11]
```
   File "/src/django-oscar-docdata/oscar_docdata/gateway.py", line 964, in __init__
     if isinstance(quantity, (int, long)):
 NameError: name 'long' is not defined

```

**Comment by jedie** [comment]
I've kept it simple in my fork and just remove "long", see: https://github.com/wearehoods/django-oscar-docdata/commit/37b65994307af8ceeef38b213e7e5e4a51a4ba86

I don't know if you still have to support Python v2 ;)

**Comment by mj026** [comment]
Yes we still have to support python 2 as long as Django 1.11 is alive (LTS release).

So to stay compatible we should change this to:

```python
if isinstance(quantity, six.integer_types): 
```

**Comment by jedie** [comment]
> Yes we still have to support python 2 as long as Django 1.11 is alive (LTS release)

That's a pity. We are completely on Python 3. Therefore my changes are only for Python 3 and i didn't have time to create a Python 2 setup ;)

**Comment by mj026** [comment]
Fixed in d1145a0b4880018c4d8dffec8ba3c9abce4bfe21

**Comment by jedie** [comment]
Another fix: https://github.com/wearehoods/django-oscar-docdata/commit/0ea0de982ee02c691f6a866f96d04372a139f9f9
